zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -361,15 +361,15 @@ mod tests {
                 time INTEGER NOT NULL,
                 sapling_tree BLOB NOT NULL
             )",
-            "CREATE TABLE received_notes (
+            "CREATE TABLE sapling_received_notes (
                 id_note INTEGER PRIMARY KEY,
                 tx INTEGER NOT NULL,
                 output_index INTEGER NOT NULL,
                 account INTEGER NOT NULL,
                 diversifier BLOB NOT NULL,
                 value INTEGER NOT NULL,
                 rcm BLOB NOT NULL,
-                nf BLOB NOT NULL UNIQUE,
+                nf BLOB,
```

Comment:
I had thought that this would be an issue, but according to the sqlite3 docs:
> SQLite treats all NULL values are different, therefore, a column with a UNIQUE constraint can have multiple NULL values.

Reply:
Fixed.